src/components/ProfileComponent/Profile.spec.js

```diff
@@ -0,0 +1,13 @@
+import {expect} from '@jest/globals';
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
```

Comment:
I would suggest to remove fireEvent, since you're not using it.

Reply:
Thanks for letting me know!  :)